Mathlib/CategoryTheory/Enriched/Ordinary/Limits/IsConicalLimit.lean

```diff
@@ -0,0 +1,41 @@
+/-
+Copyright (c) 2025 Jon Eugster. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Emily Riehl, Dagur Asgeirsson, Jon Eugster
+-/
+import Mathlib.CategoryTheory.Enriched.Ordinary.Basic
+
+/-!
+# Conical limits / enriched limits
+
+If `V` is a monoidal category and `C` a `V`-enriched ordinary category,
+a `V`-enriched limit, or "conical limit", is a limit cone `c` in `C` with
+the property that for every `X : C`, the cone obtained by applying the coyoneda
+functor `(X ⟶[V] -)` to `c` is a limit cone in `V`.
+-/
+
+universe v₁ u₁ w v' v u u'
+
+namespace CategoryTheory.Enriched
+
+open Limits
+
+variable {J : Type u₁} [Category.{v₁} J]
+variable (V : outParam <| Type u') [Category.{v'} V] [MonoidalCategory V]
```

Comment:
I think `V` should be an explicit parameter to `IsConicalLimit`. (This is what we have in the rest of the enriched category API, and I believe it should not change.)
Could you also add references to the mathematical literature? and also some minimal API (like constructors which under suitable assumptions may take as input only the `isLimit`, or only the `isConicalLimit` fields?)

Reply:
You are right, `outParam` is not needed here, only in some `class`es in a later PR. Restructrued the file.

I don't think I understand how `HasLimitsOfShape J V` could be dropped?

Reply:
We have a notion of preservation of colimits/limits which does not assume the existence of colimits/limits (https://leanprover-community.github.io/mathlib4_docs/Mathlib/CategoryTheory/Limits/Preserves/Basic.html#CategoryTheory.Limits.PreservesLimit). Then, if `eCoyoneda` preserves certains limits, we may obtain `isConicalLimit` from `isLimit`.

Reply:
- added reference.
- see other comment for continued discussion on preservation & existence of limits